modules/delivery/delivery.resolver.js

```diff
@@ -0,0 +1,31 @@
+const deliveryService = require('./delivery.service')
+const { 
+    ORDER_CREATION_FAILED,
+}= require('../../error-messages/delivery.message')
+
+const novaPoshtaQuery = {
+    getNovaPoshtaCities: async (parent, args) => await deliveryService.getNovaPoshtaCities(args.city),
+
+    getNovaPoshtaStreets: async (parent, args) => await deliveryService.getNovaPoshtaStreets(args.cityRef, args.street),
+
+    getNovaPoshtaWarehouses: async (parent, args) => await deliveryService.getNovaPoshtaWarehouses(args.city),
+
+    getNovaPoshtaPrices: async (parent, args) => await deliveryService.getNovaPoshtaPrices(args.data),
+
+    createNovaPoshtaOrder: async (parent, args) => {
```

Comment:
I don't think that it should be exposed to the GraphQL API. The delivery provider can be changed in the future so it is better to have a more unified API for this kind of operation. I think it is better to have a mutation called `submitOrder` which is going to be triggered by the client and it's going to call internally logic related to triggering delivery by particular provider depends on the data, provided in the order itself